test/test_helper.rb

```diff
@@ -338,6 +368,33 @@ def search_routing
   end
 end
 
+class Region
+  searchkick \
+    geo_shapes: {
+      territory: {tree: "quadtree", precision: "10km"}
+    }
+
+  def search_data
+    {
+      name: name,
+      text: text,
+      territory: as_geo_json
```

Comment:
If we use a virtual attribute for `territory` (see how `aisle` field is done), we can pass an object during model creation and remove the methods below.